sysutils/apcupsd/Portfile

```diff
@@ -32,6 +33,19 @@ startupitem.install         yes
 startupitem.custom_file     ${worksrcpath}/platforms/darwin/org.apcupsd.apcupsd.plist
 startupitem.type            launchd
 
+post-destroot {
+    # Move conf file to sample so it does not get overwritten
+    file rename ${destroot}${prefix}/etc/apcupsd/apcupsd.conf ${destroot}${prefix}/etc/apcupsd/apcupsd.conf.sample
+}
+
+post-activate {
+   # Create initial conf file if needed
+   if {![file exists ${prefix}/etc/apcupsd/apcupsd.conf]} {
+      file copy ${prefix}/etc/apcupsd/apcupsd.conf.sample \
+         ${prefix}/etc/apcupsd/apcupsd.conf
+   }
+}
```

Comment:
The whitespace doesn't conform to the modeline. (Indentations should be at multiples of 4 spaces.)